src/modules/ToDoList.js

```diff
@@ -0,0 +1,61 @@
+export default class ToDoList {
+  constructor() {
+    this.arr = [];
+    this.storageKey = 'todolist';
+  }
+
+  addTask() {
+    const toDoAdd = document.querySelector('#add');
+    const newItem = {
+      index: Date.now(),
+      description: toDoAdd.value,
+      completed: false,
+    };
+
```

Comment:
- [ ] As said in the project requirements, Kindly make sure the todo tasks are retrieved for an array of objects and rendered on the page when it loads. You may consider adding the arrays of tasks.
As you can see your App on it rendering :point_down: 
![image](https://user-images.githubusercontent.com/97100091/200577550-1df73f3d-bcf0-4e82-b239-561dc4c3d035.png)

Reply:
- [ ] On page load, the task array is rendered dynamically to the DOM. The tasks should appear in order of the index values for each task.